js/nixos-site.js

```diff
@@ -1,3 +1,25 @@
+$.fn.shuffleChildren = function() {
+  $.each(this.get(), function(index, el) {
+      var random = function(s = Date.now()) {
+        return function() {
+            s = Math.sin(s) * 10000;
+
+            return s - Math.floor(s);
+        };
+      };
+
+      var $el = $(el);
+      var $find = $el.children();
+
+      $find.sort(function() {
+          return 0.5 - random((new Date).getHours());
+      });
```

Comment:
This looks awfully similar to the biased [Microsoft shuffle].
The [Fisher-Yates shuffle] would be a safer bet :)

[Microsoft shuffle]: https://www.robweir.com/blog/2010/02/microsoft-random-browser-ballot.html
[Fisher-Yates shuffle]: https://en.wikipedia.org/wiki/Fisher-Yates_shuffle#The_modern_algorithm

Reply:
I just pushed a commit implementing the Fisher Yates algo.

Reply:
The seed changes at each hour, but each call also re-initialisis the sequence
it seems? That should be clealer in context:
https://github.com/NixOS/nixos-homepage/pull/1052#discussion_r1193202657